src/test/credentials/provider/ec2CredentialsProvider.test.ts

```diff
@@ -78,11 +77,6 @@ describe('Ec2CredentialsProvider', function () {
         assert.strictEqual(credentialsProvider.getDefaultRegion(), undefined)
     })
 
-    it('returns credentials', async function () {
-        const credentials = await credentialsProvider.getCredentials()
-        assert(credentials instanceof EC2MetadataCredentials)
```

Comment:
V3 credentials are no longer instances of a class, they are just plain old data instead. So this test is unnecessary.